utils/db.ts

```diff
@@ -29,6 +28,36 @@ async function getValues<T>(
   return values;
 }
 
+async function getMany<T>(
```

Comment:
```suggestion
async function getManyValues<T>(
```
To align with the other two KV-helper functions. We'd have to return an array of results instead of entries for this. It'd also be good to have a small explainer in regard to the batching functionality.

Reply:
valid point! adjusted the functionality to match the other helpers.